scaleway/resource_instance_server.go

```diff
@@ -649,12 +656,13 @@ func resourceScalewayInstanceServerUpdate(ctx context.Context, d *schema.Resourc
 		updateRequest.DynamicIPRequired = scw.BoolPtr(d.Get("enable_dynamic_ip").(bool))
 	}
 
-	volumes := map[string]*instance.VolumeTemplate{}
+	volumes := map[string]*instance.VolumeServerTemplate{}
 
 	if raw, ok := d.GetOk("additional_volume_ids"); d.HasChange("additional_volume_ids") && ok {
-		volumes["0"] = &instance.VolumeTemplate{
-			ID:   expandZonedID(d.Get("root_volume.0.volume_id")).ID,
-			Name: newRandomName("vol"), // name is ignored by the API, any name will work here
+		volumes["0"] = &instance.VolumeServerTemplate{
+			ID:      expandZonedID(d.Get("root_volume.0.volume_id")).ID,
+			Name:    newRandomName("vol"), // name is ignored by the API, any name will work here
+			Project: projectID,
```

Comment:
Maybe a problem can come from adding project here.

Reply:
Yes it is, It was removed and test are OK